Utils/FfxivWorlds.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace FFXIVVenues.Veni.Utils;
+
+public static class FfxivWorlds
+{
+
+    private static readonly Dictionary<string, bool> Regions = new()
+    {
+        { "North America", true },
+        { "Oceania", false },
+        { "Europe", true },
+        { "Japan", false },

```

Comment:
This is much simpler and means when we support another region, we can just change the region to true here and Veni will start supporting it.  🥳